test/parser_test.dart

```diff
@@ -329,14 +329,14 @@ main() {
       MAP_WITH_SOURCE_LOCATION,
       MAP_WITH_SOURCE_LOCATION_AND_NAME
     ]) {
-      var mapping = parseJson(expected);
+      var mapping = parseJson(expected) as SingleMapping;
```

Comment:
nit: why not
SingleMapping mapping = parseJson(expected);
?
Then the second "as SingleMapping" can be removed.

Reply:
Done.